pubspec.lock

```diff
@@ -71,6 +71,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.1.3"
+  clock:
```

Comment:
Nu modifica pubspec.lock într-un commit care nu are treabă cu codul (i.e. exclude-l din commit atunci când selectezi fișierele).
To fix it now, mai faci un commit unde îl modifici să fie la fel cum era înainte. If you need help, ping me.

Reply:
E ok?

Reply:
Yep